androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/PlayerControls.kt

```diff
@@ -0,0 +1,86 @@
+package com.profusion.androidenhancedvideoplayer.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.profusion.androidenhancedvideoplayer.R
+
+@Composable
+fun PlayerControls(
+    isPlaying: Boolean,
+    onPreviousClick: () -> Unit,
+    onPauseToggle: () -> Unit,
+    onNextClick: () -> Unit,
+    decoration: ControlsDecoration,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        contentAlignment = Alignment.CenterStart,
+        modifier = modifier
+            .background(decoration.backgroundColor)
+            .fillMaxSize()
+    ) {
+        Row(
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            modifier = Modifier
+                .align(Alignment.Center)
+                .fillMaxWidth()
+        ) {
+            IconButton(onClick = onPreviousClick) {
+                decoration.previousIcon()
```

Comment:
I'd rename `previousIcon` to `previousIconContent`, a common pattern to rename `content` for composables passed as arguments, i.e.: check `LazyColumn`. For me, `previousIcon` it's a better naming if it would reference the icon itself (`R.drawable.previousIcon`)